docs/staking-modules/csm/guides/mev-stealing.md

```diff
@@ -2,18 +2,18 @@
 
 ## What is MEV stealing?
 
-"MEV stealing" stands for violating the [Lido on Ethereum Block Proposer Rewards Policy](https://snapshot.org/#/lido-snapshot.eth/proposal/0x7ac2431dc0eddcad4a02ba220a19f451ab6b064a0eaef961ed386dc573722a7f). Simply speaking, any block proposed by the CSM validator with either base block rewards or MEV rewards directed to an address different from the Lido Execution Layer Rewards Vault ([Holesky](/deployed-contracts/holesky.md), [Mainnet](/deployed-contracts/index.md)) is considered to be a violating block.
+"MEV stealing" stands for violating the [Lido on Ethereum Block Proposer Rewards Policy](https://snapshot.org/#/lido-snapshot.eth/proposal/0x7ac2431dc0eddcad4a02ba220a19f451ab6b064a0eaef961ed386dc573722a7f). Simply speaking, any block proposed by the CSM validator with either base block rewards or MEV rewards directed to an address different from the Lido Execution Layer Rewards Vault ([Mainnet](/deployed-contracts/index.md), [Holesky](/deployed-contracts/holesky.md)) is considered to be a violating block.
 
 ## How does CSM react to MEV stealing?
 
-Once such a block is detected, the MEV Stealing Detection Committee (dev EOA on the testnet and CSM Multisig on the mainnet) reports this fact on-chain using the [`reportELRewardsStealingPenalty` method](https://github.com/lidofinance/community-staking-module/blob/main/docs/src/src/CSModule.sol/contract.CSModule.md#reportelrewardsstealingpenalty). Call of the method results in a lock of the stolen amount + fixed stealing fine (0.1 ETH for the testnet and most likely for the mainnet). Locked funds have not yet been burned, but will be accounted for during the required bond calculation.
+Once such a block is detected, the MEV Stealing Detection Committee (dev EOA on the testnet and CSM Multisig on the mainnet) reports this fact on-chain using the [`reportELRewardsStealingPenalty`](https://github.com/lidofinance/community-staking-module/blob/main/docs/src/src/CSModule.sol/contract.CSModule.md#reportelrewardsstealingpenalty) method. Call of the method results in a lock of the stolen amount + fixed stealing fine (0.1 ETH for the testnet and most likely for the mainnet). Locked funds have not yet been burned, but will be accounted for during the required [bond](../join-csm#bond) calculation.
```

Comment:
would prefer referencing tags or commit ids here instead of `main` branch to make it less fragile anticipating future CSM enhancements :eyes:

Reply:
Good catch!  Will do